data/json/items/archery.json

```diff
@@ -373,7 +373,7 @@
     "symbol": "(",
     "color": "yellow",
     "name": "compound bow",
-    "description": "A high-power bow with shaped cams and extra cables for high velocity shots that can be used effectively by fairly strong archers.  Currently set to a medium weight.",
+    "description": "A high-power bow with shaped cams and extra cables for high velocity shots that can be used effectively by fairly strong archers.  Currently set to a medium weight, for a balance of ease and power.",
```

Comment:
Could you please leave description as is, so translators won't need to re-translate the same text one more time?

Reply:
Ah, hadn't considered that pressure.